packages/number-field/src/NumberField.ts

```diff
@@ -155,8 +155,8 @@ export class NumberField extends TextfieldBase {
         if (value === this.value) {
             return;
         }
-        this.lastCommitedValue = value;
         const oldValue = this._value;
+        this.lastCommitedValue = oldValue;
```

Comment:
This feels contrary to what `lastCommittedValue` really means. However, I'll admit that the way the current consumption of that value works is a bit magical feeling...maybe there's something better to be done with that property.